Jenkinsfile

```diff
@@ -47,30 +67,64 @@ pipeline {
         timeout(time: 2, unit: 'HOURS')
     }
 
+    parameters {
+        string(
+            name: 'CMAKE_UTILS_REFERENCE',
+            description: '''
+                rticommunity/rticonnextdds-cmake-utils repository reference to use (Branch or PR).
+                 E.g.: PR-123, release/7.3.0, master
+            ''',
+            defaultValue: '',
+            trim: true,
+        )
+    }
+
     stages {
         stage('Build sequence') {
             agent {
-                dockerfile {
-                    filename 'resources/docker/Dockerfile.linux'
-                    label 'docker'
-                }
+                label "${nodeManager.labelFromJobName()}"
             }
 
             environment {
                 RTI_INSTALLATION_PATH = "${env.WORKSPACE}"
+                VIRTUAL_ENV = "${env.WORKSPACE}/.venv"
             }
 
             stages {
+                stage('Select CMake utils version') {
+                    steps {
+                        script {
```

Comment:
If we could get rid of the script blocks that would be great :)

Reply:
It does not like that:
```
[GitHub Checks] GitHub check (name: Validation (Linux), status: in_progress) has been published.
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 75: Method calls on objects not allowed outside "script" blocks. @ line 75, column 17.
                   nodeManager.runInsideExecutor() {
                   ^

WorkflowScript: 107: Method calls on objects not allowed outside "script" blocks. @ line 107, column 29.
                               nodeManager.runInsideExecutor() {
                               ^

WorkflowScript: 118: Method calls on objects not allowed outside "script" blocks. @ line 118, column 17.
                   nodeManager.runInsideExecutor() {
                   ^

3 errors
```